packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.routing.js

```diff
@@ -196,8 +197,18 @@ export default /* @ngInject */ ($stateProvider) => {
               'app.dedicated.server.dashboard.bandwidth-public-order',
               { productId: serverName },
             ),
-      serviceMonitoring: /* @ngInject */ ($stateParams, Server) =>
-        Server.getAllServiceMonitoring($stateParams.productId),
+      serviceMonitoring: /* @ngInject */ ($transition$, Server) =>
+        Server.getAllServiceMonitoring($transition$.params().productId),
+      technicalDetails: /* @ngInject */ ($http, $transition$) =>
+        $http
+          .get(
+            `/dedicated/technical-details/${$transition$.params().productId}`,
```

Comment:
As mentioned previously (https://github.com/ovh/manager/pull/4313#discussion_r566680560) maybe we could inject `serverName` instead as it is already declared